data/functions/utils.sql

```diff
@@ -5,6 +5,15 @@ begin
 end
 $$ language plpgsql immutable;
 
+create or replace function get_transaction_year(transaction_date timestamp, report_year numeric)
```

Comment:
Indeed it can:  https://www.postgresql.org/docs/current/static/xfunc-overload.html